src/views/Languages/index.jsx

```diff
@@ -111,10 +117,9 @@ const tagReposMapping = repositories =>
 )
 export default class Languages extends Component {
   state = {
-    error: null,
-    // we need a way to know when github loading is done.
-    // github results require calling fetchMore possibly many times.
-    githubLoading: false,
+    hasNextPage: true,
+    isNextPageLoading: false,
+    items: [],
```

Comment:
Can we remove this state? Based on what I see, it seems that we can obtain this from props. Thus by using state, there seem to be 2 sources of truth, state & props. I think it is better if we do as before, obtaining the items by combining `githubData` and `bugzillaData` in the render method. What do you think?